src/panels/lovelace/cards/hui-column-card.js

```diff
@@ -0,0 +1,71 @@
+import createCardElement from '../common/create-card-element.js';
+import createErrorConfig from '../common/create-error-card-config.js';
+
+class HuiColumnCard extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: 'open' });
+    this._elements = [];
+  }
+
+  getCardSize() {
+    let totalSize = 0;
+    this._elements.forEach((element) => {
+      totalSize += typeof element.getCardSize === 'function' ?
+        element.getCardSize() : 1;
+    });
+    return totalSize;
+  }
+
+  set config(config) {
+    this._elements = [];
+    const root = this.shadowRoot;
+
+    while (root.lastChild) {
+      root.removeChild(root.lastChild);
+    }
+
+    if (config && config.cards && Array.isArray(config.cards)) {
```

Comment:
Make it a guard clause. Check for negative case, add error element and then return.